tools/v8_gypfiles/v8.gyp

```diff
@@ -1884,6 +1886,54 @@
       },
     },  # postmortem-metadata
 
+    {
+      'target_name': 'v8_zlib_adler32_simd',
+      'type': 'static_library',
+      'conditions': [
+        ['target_arch in "ia32 x64" and OS!="ios"', {
+          'defines': [ 'ADLER32_SIMD_SSSE3' ],
+          'conditions': [
+            ['OS=="win"', {
+              'defines': [ 'X86_WINDOWS' ],
+            },{
+              'defines': [ 'X86_NOT_WINDOWS' ],
+            }],
+            ['OS!="win" or llvm_version!="0.0"', {
+              'cflags': [ '-mssse3' ],
+              'xcode_settings': {
+                'OTHER_CFLAGS': [ '-mssse3' ],
+              },
+            }],
+          ],
+        }],
```

Comment:
It's pretty unclear why it would be safe to assume SSSE3 support here.

Reply:
this is copied from `deps/zlib/zlib.gyp`; runtime CPU detection is used to decide which SIMD version to call into. (Or are you suggesting that we should add a comment to that effect?)